.github/workflows/auto-commits.yml

```diff
@@ -0,0 +1,70 @@
+name: auto-commits
+
+on:
+  push:
```

Comment:
Ok and it will not "create" new commits but will update a done commit right /

Reply:
It will create commits and push it to PR branches, but also to master. It will not touch commits on PRs but add new ones on top of it. If a person it very quick and merges before this PR runs on PR, it will directly update master.

Reply:
I'm not 100% sure whether the new commits trigger CI. Github will not trigger CI if the trigger does not have a GH action (like GH actions). It's possible that it will author the new commits with the original committer (see my icon next to the autogenerated commit). A limitation that no CI is triggered is also not mentioned in the actions repo (just that action matrices do not work). Otherwise we would end up with something similar like https://github.com/KhronosGroup/Vulkan-Hpp/pull/1531, we could fix it by just running this workflow on master.

Reply:
Ah ok, due to you ricon being next to it i thought it was a commit from you that was edited by the bot. In general i like having this automated and i trust you with knowing the est way to do it.

Reply:
We can merge this right ?

Reply:
Yes. The last commit on the stub file was done by the bot. It demos the change